### DISCUSSION TITLE
[Feature Proposal]: Add a "border-spacing" utility

### DISCUSSION THREAD

**Discussion by nicholasnet** [tailwindlabs/tailwindcss#1437] [Ideas]
Can we add `border-spacing-0` class. Since without using `border-spacing: 0` its hard to create table with round border.

**Comment by retrostar** [comment]
Just linking some comment of some former issue which clarifies this a bit more:
https://github.com/tailwindlabs/tailwindcss/issues/380#issuecomment-745443458
